src/main/java/com/sap/sgs/phosphor/fosstars/data/github/HasSecurityTeam.java

```diff
@@ -39,9 +39,6 @@ protected Feature supportedFeature() {
   @Override
   protected Value fetchValueFor(GitHubProject project) throws IOException {
     logger.info("Figuring out if the project has a security team ...");
-    if (securityTeam.existsFor(project.url())) {
-      return HAS_SECURITY_TEAM.value(true);
-    }
-    return HAS_SECURITY_TEAM.unknown();
```

Comment:
I think this should be `unknown` right?

As we check this feature based on a file which we update on a regular basis. If the security team entry is not present for a project in the file. It doesn't mean that we can confidently say that the project doesn't have a security team. It would be `unknown` right?

Please correct me if I am mistaken.

Reply:
You are absolutely right. However, returning an unknown value makes overall confidence of rating lower. I am not sure if it is worth it. See #225